content/docs/components/animated-shiny-text.mdx

```diff
@@ -61,6 +63,16 @@ Add the following animations to your global CSS file inside the `@theme inline`
 
 </Tabs>
 
+## Usage
+
+```tsx
+import { AnimatedShinyText } from "@/components/magicui/animated-shiny-text";
+```
+
+```tsx
+<AnimatedShinyText>{children}</AnimatedShinyText>
```

Comment:
You should give an example text

`<AnimatedShinyText>Hello World</AnimatedShinyText>`

Reply:
`childrens` are updated with text.